src/redux/actions/differentialExpression/loadDifferentialExpression.js

```diff
@@ -8,12 +8,16 @@ import getTimeoutForWorkerTask from 'utils/getTimeoutForWorkerTask';
 const getCellSetName = (name) => (name?.split('/')[1] || name);
 
 const loadDifferentialExpression = (
-  experimentId, cellSets, comparisonType, tableState,
+  experimentId, cellSets, comparisonType, tableState, newAdvancedFilters = null,
 ) => async (dispatch, getState) => {
+  const advancedFilters = newAdvancedFilters
```

Comment:
`||` is for bools, I know it works too because #javascript, but I would say instead use `??` which is for null/undefined.